documentation/tutorial/01_maps_and_tiles.md

```diff
@@ -0,0 +1,157 @@
+the `ǂ` character. This will provide a bit more visual context to the player of where
+they are on the map.
+
+So now let's say we want to customize this Grass tile a bit further by creating a
+grass tile that is not passable. We could create a new class called something like
+`BlockedGrass`, or we could further customize our existing `Grass` class. Let's go
+with the latter for now.
+
+```
+class Grass < Goby::Tile
+  def initialize(passable: true, seen: false, description: "", events: [], monsters: [])
+    @passable = passable
+    @seen = seen
+    @description = description
+    @events = events
+    @monsters = monsters
+    @graphic = set_graphic(passable)
+  end
+
+  private
+
+  def default_graphic
+    return "ǂ"
+  end
+
+  def set_graphic(passable)
+    passable ? default_graphic : "¥"
+  end
+end
+```
+
+So here what we've done is added another private method called `#set_graphic`.
+This method is responsible for setting the `@graphic` property of the `Grass`
+instance. When an instance is created that is passable, the "ǂ" character will
+represent that tile, but when an instance is created that is not passable, the "¥"
+character will be displayed instead.
+
+## Map Class
+
+So now we arrive at the point in which we want to assemble our tiles into something
+cohesive. This is the responsibility of the `Map` class. The `Map` class is quite
+simple, with just 3 properties and 3 methods. The properties are:
+
+- name (defaults to "Map")
+- tiles (defaults to [[Tile.new]])
+- music (defaults to nil)
+
+For now, we will ignore the methods as they aren't something that you will be
+actively working with at this point.
+
+Now that we know the properties of the `Map` class, we are ready to start
+constructing our own maps. Let's start by creating a map called `Meadow1`.
+
+```
+class Meadow1 < Map
+  def initialize
+    super(name: "Open Meadow")
+
+    # Define the main tiles on this map.
+    grass = Grass.new(description: "You are standing on some grass.")
+    blocked_grass = Grass.new(passable: false)
+
+    # Fill the map with "grass."
+    @tiles = Array.new(9)
+
+    normal_row = [grass, grass, grass, grass, grass]
+    blocked_row = [grass, blocked_grass, blocked_grass, blocked_grass, grass]
+
+    @tiles[0] = normal_row
+    @tiles[1] = normal_row
+    @tiles[2] = normal_row
+    @tiles[3] = blocked_row
+    @tiles[4] = blocked_row
+    @tiles[5] = blocked_row
+    @tiles[6] = normal_row
+    @tiles[7] = normal_row
+    @tiles[8] = normal_row
+
+  end
+end
+```
+
+When we create an instance of `Meadow1`, what we end up with is a map that has
+passable grass on its borders, and blocked grass in the middle, as a result of
+adding tiles to the `@tiles` property in a specific order.
+
+## Running this simple project
+
+Now that we have tiles and a map defined, we are ready to lauch this simple game.
+When you initialized your Goby project by calling `goby` at the command line, it
```

Comment:
You can probably come back to this later, but this is the first time you're telling them to initialize the project. The verbiage `When you initialized your Goby project...` thus sounds a little confusing.